src/encoded/audit/experiment.py

```diff
@@ -2475,6 +2475,20 @@ def audit_experiment_target(value, system, excluded_types):
     if 'control' in target['investigated_as']:
         return
 
+    # Experiment target should be untagged
+    non_tag_mods = ['Methylation',
+                    'Monomethylation',
+                    'Dimethylation',
+                    'Trimethylation',
+                    'Acetylation',
+                    'Ubiquitination',
+                    'Phosphorylation']
+    if any(mod['modification'] not in non_tag_mods
```

Comment:
for "audit experiment target" the category should be also something like "inconsistent experiment target" instead of "tagged target"